clipper_admin/clipper_admin/kubernetes/kubernetes_container_manager.py

```diff
@@ -277,46 +297,46 @@ def connect(self):
 
         if len(external_node_hosts) == 0 and (self.useInternalIP):
             msg = "No external node addresses found. Using Internal IP address"
-            logger.warn(msg)
+            self.logger.warn(msg)
             for addr in node.status.addresses:
                 if addr.type == "InternalIP":
                     external_node_hosts.append(addr.address)
 
         if len(external_node_hosts) == 0:
-            msg = "Error connecting to Kubernetes cluster. No external node addresses found"
-            logger.error(msg)
+            msg = "Error connecting to Kubernetes cluster. No external node addresses found. If you are running Kubernetes locally, you can pass in KubernetesContainerManager(useInternalIP=True) to connect to local Kubernetes cluster"
```

Comment:
You don't need to be running Kubernetes locally to use internal ip addresses. Remove that part of the error message.